python/src/aiconfig/Config.py

```diff
@@ -261,6 +261,9 @@ async def run(
         model_name = self.get_model_name(prompt_data)
         model_provider = AIConfigRuntime.get_model_parser(model_name)
 
+        # Clear previous run outputs if they exist
+        self.delete_output(prompt_name)
```

Comment:
Does this remove the output from the config?

Reply:
Yes `delete_output` deletes the outputs and returns them. Typescript does the same with `deleteOutput`
py: https://github.com/lastmile-ai/aiconfig/blob/main/python/src/aiconfig/schema.py#L530